text-recognition/final/text-recognition/Podfile

```diff
@@ -1,11 +1,12 @@
 # Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '11.0'
```

Comment:
do we need this? I believe MLkit supports iOS 9. We don't need to limit the min version to 11